assets/js/linear_algebra/dot_product_symmetric.js

```diff
@@ -168,36 +127,77 @@ function plot(scatter, axis, tt){
+  
+  let [lines_vector_u, texts_vector_u] = lib.text_matrix_to_list(
+          [[u.coord.x.toFixed(2), u.coord.y.toFixed(2), u.coord.z.toFixed(2)]],
+          [last_col_coord, start_coord_y],
+          14, [['xu', 'yu', 'zu']], 5
+          ),
+      [lines_vector_v, texts_vector_v] = lib.text_matrix_to_list(
+          [[v.coord.x.toFixed(2), v.coord.y.toFixed(2), v.coord.z.toFixed(2)]],
+          [last_col_coord, start_coord_y],
+          14, [['xv', 'yv', 'zv']], 15
+          ),
+      [lines_uT, texts_uT] = lib.text_matrix_to_list(
+          [[u.coord.x.toFixed(2)], [u.coord.y.toFixed(2)], [u.coord.z.toFixed(2)]],
+          [start_coord_x, last_row_coord],
+          14, [['xu'], ['yu'], ['zu']], 5
+          ),
+      [lines_vT, texts_vT] = lib.text_matrix_to_list(
+          [[v.coord.x.toFixed(2)], [v.coord.y.toFixed(2)], [v.coord.z.toFixed(2)]],
+          [start_coord_x, last_row_coord],
+          14, [['xv'], ['yv'], ['zv']], 15
+          );
+
+  let uTv_texts = [],
+      uTv_lines = [],
+      vTu_texts = [],
+      vTu_lines = [];
+
+
+  uTv_texts.push(...texts_uT);
+  uTv_texts.push(...texts_vector_v);
+  uTv_texts.push(uT_name, v_name);
+
+  uTv_lines.push(...lines_uT);
+  uTv_lines.push(...lines_vector_v);
+
+  vTu_texts.push(...texts_vT);
+  vTu_texts.push(...texts_vector_u);
+  vTu_texts.push(vT_name, u_name);
+
+  vTu_lines.push(...lines_vT);
+  vTu_lines.push(...lines_vector_u);
+
+  console.log(uTv_lines, vTu_lines);
 
   if (position_state == 0) {
-    lib.plot_texts(lib.text_table_to_list(
-      uTv_text_table,
-      start_coord_x=start_coord_x, start_coord_y=start_coord_y,
-      w_unit=w_unit, h_unit=h_unit,
-      dws_array=dws_array, dhs_array=dhs_array),
-    tt=1000, name='transition');
-
+    lib.plot_texts(uTv_texts, tt, 'transition');
+    lib.plot_lines(uTv_lines, tt, 'bracket');
   } else {
-    lib.plot_texts(lib.text_table_to_list(
-      vTu_text_table,
-      start_coord_x=start_coord_x, start_coord_y=start_coord_y,
-      w_unit=w_unit, h_unit=h_unit,
-      dws_array=dws_array, dhs_array=dhs_array),
-    tt=1000, name='transition');
-  };
+    lib.plot_texts(vTu_texts, tt, 'transition');
+    lib.plot_lines(vTu_lines, tt, 'bracket');
+  }
+
+  let dot_product_texts_at_bot = [
+      {text: uTv.toFixed(3),
+       x: last_col_coord,
+       y: last_row_coord + 0.05 * h_unit,
+       text_color: 0, text_opacity: 1,
+       font_size: 14, tt: 0, key: 'text_at_bot'}
+  ];
 
-  let uTv_texts = [     
+  lib.plot_texts(dot_product_texts_at_bot, tt, 'text_at_bot');
+
+  let dot_product_texts = [     
     {text:'u\u1d40v = '.concat(uTv.toFixed(3)),
      x: (v.x * uTv/2).toFixed(2),
      y: (v.y * uTv/2).toFixed(2),
      text_color: 0, text_opacity: 1,
-     font_size: 15, tt: 0, key: 'uTv_texts'}
+     font_size: 15, tt: 0, key: 'texts_in_blue_line'}
   ];
   
-  lib.plot_texts(uTv_texts, tt=0, name='uTv_texts');
+  lib.plot_texts(dot_product_texts, tt=0, name='texts_in_blue_line');
```

Comment:
double check for values of `tt` here, is it `0` or `tt`?

Reply:
done